src/autotype/AutoType.cpp

```diff
@@ -390,7 +390,7 @@ bool AutoType::parseActions(const QString& actionSequence, const Entry* entry, Q
 {
     QString tmpl;
     bool inTmpl = false;
-    m_autoTypeDelay = qMax(config()->get("AutoTypeDelay").toInt(), 0);
+    m_autoTypeDelay = qMax(config()->get(Config::AutoTypeDelay).toInt(), 0);
```

Comment:
@phoerious wouldn't it make sense to validate that the delay is `> 0` from within the config?

Reply:
I don't think so. That would be too much case-specific logic in such a generic place and I don't really want to change too much of what's not strictly part of the configuration.

Reply:
This is appropriate IMO.